### DISCUSSION TITLE
角色的武器是有等级的

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#657] [Ideas]
好像记得某一天你问过，我看了下数据库是有的 @Mark9804

**Comment by mark9804** [comment]
#388 ？我印象里一直是知道 `米游社` 有这个数据的

**Reply by mark9804** [reply]
对哦，`我的` 忘记把角色等级和武器等级加上了

**Reply by Arondight** [reply]
米游社命令的也没有，只有精炼等级，没有强化等级

**Reply by mark9804** [reply]
阿哲，我都没发现没有展示武器等级，既然没有人来提 feature request ，那就再摆一阵吧

**Reply by Arondight** [reply]
嗯，我就是突然想起来可能某一天跟你讨论过这个然后我说错了，所以才发了个帖子
